extensions/amp-story/1.0/test/validator-amp-story-parallax-fx.html

```diff
@@ -0,0 +1,112 @@
+<!--
```

Comment:
Can we split the validation into a separate PR?  I think we might want to experiment with publisher opt-in vs. user opt-in (via, like, a button), so I don't think the API is final yet here

Reply:
Even though this is gated behind an experiment flag?

Reply:
Hmm, I guess that's ok 👍